front/src/views/routes/workspace/admin/database/DatabaseTable.vue

```diff
@@ -494,27 +537,39 @@ export default {
       this.resetColumnEdit()
       this.resetSecondarySources()
     },
+    /**
+     * Retrieve records according to page wanted
+     */
     onUpdateContent (pageIndexToGo) {
       this.currentPageIndex = pageIndexToGo
       this.loadCurrentTableData()
     },
+    /**
+     * Retrieve view with action and column
+     */
     async loadTableAndProcess () {
       this.block.loading = true
       this.block.content = {
         total: 0,
         data: null,
       }
+      // Legacy: doesn't use anymore
```

Comment:
do you think we need to add a `TODO: ` ?

Reply:
I think we will remove it in the new version. So I don't think a "Todo" it's necessary.